examples/demo/site/config/modules/login/account_activation.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html lang="en">
+	<head>
+	  <meta charset="utf-8">
+	  <title>Activation</title>
+	  <meta name="description" content="Activation">
+	  <meta name="author" content="ROC CMS">
+	</head>
+
+	<body>
+		<p>Thank you for registering at <a href="$host">ROC CMS</a></p>
```

Comment:
I don't remember if we have the site's name in configuration
but I guess this "ROC CMS"  should be customizable.

question 2, why not using smarty template here?
(just a question, not a suggestion)